rest-api-spec/src/main/resources/rest-api-spec/api/ml.validate_detector.json

```diff
@@ -1,7 +1,7 @@
 {
   "ml.validate_detector": {
     "documentation": {
-      "url": "https://www.elastic.co/guide/en/machine-learning/current/ml-jobs.html",
+      "url": null,
```

Comment:
does https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ml-put-job work here?

Reply:
Not really, `ml.validate_detector` is related but different and private.